.erb-lint.yml

```diff
@@ -0,0 +1,40 @@
+---
+EnableDefaultLinters: true
+linters:
+  DeprecatedClasses:
+    enabled: true
+    addendum: "Reference: https://sarken.github.io/docs/front_end_coding/classes.html."
```

Comment:
Should we link to this yet?

Reply:
No, that's just a temporary home for it while it's in progress -- the goal is to eventually put it under otwcode.

Reply:
I'll remove it for now.